### DISCUSSION TITLE
Behold, Doodles in Space

### DISCUSSION THREAD

**Discussion by jmdelvecchio** [Doodleverse/s2m_engine#4] [Show and tell]
Nothing like a proposal deadline to get you to write code:
![image](https://user-images.githubusercontent.com/17852785/205353640-bc11a87f-7390-4e6f-813b-667bc7b09776.png)

I exploited the fact that when you convert the GeoTIFFS from Earth Engine to jpegs it creates a `jpg.aux.xml` file with the spatial data, so I just parsed file names to match labels to xmls. 

I wrote meself [a Python module](https://github.com/jmdelvecchio/PyCoGSS/blob/main/pycogss/pycogss/tiff_tools.py) so I could run everything from the command line, but then I got lazy and stole yalls popup dialogs for directory selection. 

Don't read the code too closely!!!

Most important thing for me is that "NoData" comes up as class 0 but there is also a class assigned to 0.

**Comment by ddbuscombe** [comment]
This looks great! When you have time, do you think you could put together a small example of inputs and outputs and how to call the functions? Or if you like, please go ahead and make a folder inside https://github.com/Doodleverse/s2m_engine/tree/main/s2m_engine called 'jpg2geotif' or something and add a function, then make a PR (perhaps with an example workflow :pray: ).

**Reply by jmdelvecchio** [reply]
OK this is long overdue (so sorry, but also why would editors put manuscript deadlines at the end of December??) but this is on my to-do list this week into next. Be on the lookout.

**Reply by ddbuscombe** [reply]
Sounds great! No time-pressure! :pray:
